src/main/java/it/gov/pagopa/merchant/service/MerchantServiceImpl.java

```diff
@@ -1,8 +1,40 @@
 package it.gov.pagopa.merchant.service;
 
+import it.gov.pagopa.merchant.constants.MerchantConstants;
+import it.gov.pagopa.merchant.dto.InitiativeDTO;
+import it.gov.pagopa.merchant.dto.mapper.Initiative2InitiativeDTOMapper;
+import it.gov.pagopa.merchant.exception.MerchantException;
+import it.gov.pagopa.merchant.model.Merchant;
+import it.gov.pagopa.merchant.repository.MerchantRepository;
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
 
+import java.util.List;
+
 @Service
 public class MerchantServiceImpl implements MerchantService {
```

Comment:
Do we want to have a service structure similar to idpay-payment instead of having a single MerchantService?